### DISCUSSION TITLE
How to use Pipeline with UnifiedJedis client

### DISCUSSION THREAD

**Discussion by Datta19** [xetorthio/jedis#3491] [Q&A] [answered]
If we use pipeline  with jedis then it works fine but same thing is not working with UnifiedJedis. Is there any way to use pipeline with UnifiedJedis?

Pipeline  pipeline1 =jedis.pipelined();
Pipeline pipeline =  client.pipelined(); // Here compiler gives compile time error "Type mismatch: cannot convert from Object to Pipeline"

**Comment by sazzad16** [comment, marked as answer]
You are required to do a type-cast:
```
Pipeline pipeline = (Pipeline) client.pipelined();
```

From Jedis 5, you would be able to do:
```
PipelineBase pipeline = client.pipelined();
```

**Comment by Datta19** [comment]
@sazzad16 Thanks for the valuable response.
